misc/2017-09-20-ray-dillinger-if-id-known.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ray Dillinger -  If I'd Known What We Were Starting (2017-09-20)
+translation_by: Julien Urraca
+categories: ['chaincode-curriculum']
+tags: ['history']
+---
+
+Autor: Ray Dillinger
+Texto original: [link](https://www.linkedin.com/pulse/id-known-what-we-were-starting-ray-dillinger/)
```

Comment:
It would be more consistent to add a a link like `<https://www.linkedin.com/pulse/id-known-what-we-were-starting-ray-dillinger/>` rather than [link]